.gitignore

```diff
@@ -1 +1,3 @@
 node_modules
+es
+lib
```

Comment:
Transient dirs should not be in the gitignore, they should be removed once the task that needed them finishes. NPM offers the super handy `pre...` and `post...` tasks for this, so if a dir `bar` is used by task `foo`, you can write `"postfoo": "shx rm -rf bar"` and cleanup happens automatically when you `npm run foo`.